observability/logging/cluster-logging-support.adoc

```diff
@@ -10,20 +10,21 @@ include::snippets/logging-supported-config-snip.adoc[]
 include::snippets/logging-compatibility-snip.adoc[]
 include::snippets/log6x-loki-statement-snip.adoc[]
 
-{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. You can use it to forward logs to various supported systems.
 
 {logging-uc} is not:
 
 * A high scale log collection system
 * Security Information and Event Monitoring (SIEM) compliant
+* A "bring your own" (BYO) log collector configuration
```

Comment:
In WMCO docs, we use [Bring-Your-Own (BYO)](https://90136--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/byoh-windows-instance). Should we be consistent?